src/components/VisuallyHidden.tsx

```diff
@@ -1,43 +1,22 @@
-import React from "react"
-import styled from "@emotion/styled"
+import React, { useContext } from "react"
+import { VisuallyHidden as ChakraVisuallyHidden } from "@chakra-ui/react"
 
-// "Accessibility/SEO Friendly CSS Hiding"
-// Source: https://css-tricks.com/snippets/css/accessibilityseo-friendly-css-hiding/
-const Container = styled.span`
-  border: 0;
-  clip: rect(1px, 1px, 1px, 1px);
-  clip-path: inset(50%);
-  height: 1px;
-  margin: -1px;
-  overflow: hidden;
-  padding: 0;
-  position: absolute;
-  width: 1px;
-  word-wrap: normal !important;
-  &:focus {
-    background-color: #eeeeee;
-    clip: auto !important;
-    clip-path: none;
-    color: #444444;
-    display: block;
-    font-size: 1em;
-    height: auto;
-    left: 5px;
-    line-height: normal;
-    padding: 15px 23px 14px;
-    text-decoration: none;
-    top: 5px;
-    width: auto;
-    z-index: 100000;
-  }
-`
+import { ZenModeContext } from "../contexts/ZenModeContext"
+
+// Todo: Refactor to from isHidden boolean
 
 export interface IProps {
   children?: React.ReactNode
-  isHidden?: boolean
 }
 
-const VisuallyHidden: React.FC<IProps> = ({ isHidden = false, children }) =>
-  isHidden ? <Container>{children}</Container> : <>{children}</>
+const VisuallyHidden: React.FC<IProps> = ({ children }) => {
```

Comment:
I think we need to change the name of this component. This is not a generic component anymore. This is strictly related to zen mode. Would suggest calling this component `ZenMode`.

Reply:
Agree... just pushed a commit for this. I think this should be good to go now.